src/components/screens/signMultiSignTransaction/reviewSign/reviewSign.js

```diff
@@ -56,37 +54,6 @@ const flattenTransaction = ({ moduleAssetId, asset, ...rest }) => {
   return transaction;
 };
 
-const getNumbersOfSignaturesRequired = ({ keys, isGroupRegistration }) => {
```

Comment:
Why were these removed? I added this as a fix in https://github.com/LiskHQ/lisk-desktop/pull/3579